app/score_calculator/yaku_check/blocks_yaku_checker.py

```diff
@@ -63,6 +64,20 @@ def two_blocks_checker(self) -> Yaku:
             result = Yaku.TwoTerminalChows
         return result
 
+    def four_blocks_checker(self) -> Yaku:
+        result: Yaku = Yaku.ERROR
+        if self.is_big_four_winds:
+            result = Yaku.BigFourWinds
+        elif self.is_little_four_winds:
+            result = Yaku.LittleFourWinds
+        elif self.is_quadruple_chow:
+            result = Yaku.QuadrupleChow
+        elif self.is_four_pure_shifted_chows:
+            result = Yaku.FourPureShiftedChows
+        elif self.is_four_pure_shifted_pungs:
+            result = Yaku.FourPureShiftedPungs

```

Comment:
이런건 형태는 dict를 만들거나, match/case 문을 사용하는게 일반적인 practice 입니다.

Reply:
아 죄송합니다 코드를 착각했네요. tuple list로 만들어서 다음과 같은 형태로 보통 사용합니다.
```suggestion
        conditions = [(self.is_big_four_winds,  Yaku.BigFourWinds), ...]
        yaku = next((value for checker, value in conditions if checker()), Yaku.ERROR)

```

Reply:
오 깔끔하네요

Reply:
일반적으로 이런 형태의 코드가 있을때, 데이터 부분과 로직 부분을 어떻게 하면 분리할수 있을지를 고민해보시면
코드를 깔끔한 형태로 짜기 좋습니다.